src/app/components/countriesCard/countryList.tsx

```diff
@@ -0,0 +1,27 @@
+"use client"
+import CountryCard from "@/app/components/countriesCard/Card";
+import { Country } from "@/app/components/countriesCard/fetchCountriesData"; 
+
```

Comment:
Hello. Please avoid importing from the root as this causes circular dependencies.. Use relative imports with '../../'

Reply:
Okay d'accord